dist/preview release/babylon.canvas2d.d.ts

```diff
@@ -406,6 +406,7 @@ declare module BABYLON {
          * When enabled the Prim2DBase.pointerEventObservable property will notified when appropriate events occur
          */
         interactionEnabled: boolean;
+        fitRenderingDevice: boolean;
```

Comment:
you were not supposed to add the d.ts I think it was a mistake but that's fine this time the impact is not negative